pkgs/applications/misc/octoprint/default.nix

```
@@ -202,6 +203,7 @@ let
               passthru = {
                 python = self.python;
                 updateScript = nix-update-script { };
+                tests = nixosTests.octoprint;
```

passthru.tests is expected to be an attrset.
```suggestion
                tests = {
                  inherit (nixosTests) octoprint;
                };
```